sdk/servicebus/azservicebus/README.md

```diff
@@ -0,0 +1,50 @@
+# Microsoft Azure Service Bus Client for Golang
+[![Go Report Card](https://goreportcard.com/badge/github.com/Azure/azure-service-bus-go)](https://goreportcard.com/report/github.com/Azure/azure-service-bus-go)
+[![godoc](https://godoc.org/github.com/Azure/azure-service-bus-go?status.svg)](https://godoc.org/github.com/Azure/azure-service-bus-go)
+[![Build Status](https://travis-ci.org/Azure/azure-service-bus-go.svg?branch=master)](https://travis-ci.org/Azure/azure-service-bus-go)
+[![Coverage Status](https://coveralls.io/repos/github/Azure/azure-service-bus-go/badge.svg?branch=master)](https://coveralls.io/github/Azure/azure-service-bus-go?branch=master)
```

Comment:
These need to be updated once this is merged.

Reply:
It looks like this is the only package in the repo that's like this, which makes me think I should just remove them altogether.